etsin_finder/resources/download_resources.py

```diff
@@ -23,8 +24,10 @@
 
 from etsin_finder.utils.log_utils import log_request
 
+PACKAGE_SIZE_LIMIT = 10000000
```

Comment:
```suggestion
PACKAGE_SIZE_LIMIT = 5 * 1024 ** 4
```
One way could be to put this into constants.py and supply it to the frontend with the `/api/app_config` endpoint so the values would be in sync between frontend and backend. But I'm okay with the value being duplicated here.